manim/animation/creation.py

```diff
@@ -81,8 +83,10 @@ def construct(self):
 if TYPE_CHECKING:
     from manim.mobject.text.text_mobject import Text
 
+from manim.constants import RIGHT
```

Comment:
I like using this over the `from ..constants import TAU` a few lines below, would you mind changing this to
```py
from manim.constants import RIGHT, TAU
```
and removing the 
```py
from ..constants import TAU
```

Reply:
Got it.